### PR TITLE
:Adding timeouts to public backends.

### PR DESCRIPTION
**Pull request by carlosroman**
Also added the same to our neo4j sidekicks.

### CODE REVIEW

**Comment by catalin-ilea** [comment]
Could you please add the same for the rest of the public apis and api-component-policy?

**Comment by Zapadlo** [comment]
We don't have a default value in vcb then?

+1

**Comment by catalin-ilea** [comment]
We should have this in VCB from my point of view. VCB supports adding public frontends but I guess we forgot about that when we implemented the pattern in the sidekicks

File: public-people-api-sidekick@.service [hunk 1]
```diff
@@ -11,6 +11,7 @@ ExecStart=/bin/sh -c "\
   etcdctl mkdir /ft/services/$SERVICE/servers; \
   etcdctl set /ft/healthcheck/$SERVICE-%i/path /__health; \
   etcdctl set /ft/healthcheck/$SERVICE-%i/categories read; \
+  etcdctl set /vulcand/backends/public-services-people/backend \'{\"Type\": \"http\", \"Settings\": {\"KeepAlive\": {\"MaxIdleConnsPerHost\": 256, \"Period\": \"35s\"}}}\'; \
```
Comment (catalin-ilea) [inline public-people-api-sidekick@.service]: I don't think this does what is supposed to do. These settings should be applied to backend vcb-public-people-api

Comment (catalin-ilea) [inline public-people-api-sidekick@.service]: We need to clean up these sidekicks and configure this in vulcan-config-builder

**Pull request merged**